Gems/Atom/RPI/Code/Source/Platform/Common/MSVC/atom_rpi_public_msvc.cmake

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+ly_add_source_properties(
+    SOURCES External/MaskedOcclusionCulling/MaskedOcclusionCullingAVX2.cpp
+    PROPERTY COMPILE_OPTIONS
+    VALUES /arch:AVX2 /W3
```

Comment:
this is lowering the warning level, should be fixed instead or wrapped the warning in the place where it is triggered

Reply:
Oh, I just see it was moved from another place, will create a ticket to myself to fix this...

Reply:
Also note that this is for "External" code, original source is https://github.com/GameTechDev/MaskedOcclusionCulling

Reply:
ah, gotcha, yeah, this should be moved to the 3p system so it is included with the proper `/external:I` or `isystem`

Reply:
I agree with @burelc-amzn here, since this is 3rdParty code, we would want to avoid modifying it if we can, to make future updates easier to take.

I think a nice compromise is to report the warning along with a potential fix to the MaskedOcclusionCulling repo as an issue to the GameTechDev team

Reply:
Fine by me, I just dont want to encourage changing warnings across our source code. This 3rdParty also looks like it should be in 3p and not compiled as part of our code.